docs/python/editing.md

```diff
@@ -67,11 +72,48 @@ For help with common Intellisense and Python editing issues, check the table bel
 | Problem | Cause | Solution |
 | --- | --- | ---|
 | The path to the Python interpreter is incorrect | The Python extension tries to find and select what it deems the best environment for the workspace. | Make sure you selected a valid interpreter path by running the **Python: Select Interpreter** command (see [Environments](/docs/python/environments.md)). |
-| Import could not be found *(reportMissingImports)* | The package you are trying to import is not installed in the selected Python environment. | First, ensure the package is installed in your environment. If working in a `.py` file, type `python -m pip install {package_name}` in an activated terminal to install a package into your environment. For a Notebook, type `%pip install {package name}` in your code cell. If the package is installed, ensure you are using the correct environment/interpreter that matches the environment activated in your terminal. To select a new Python interpreter, use the **Python: Select Interpreter** command.|
+| Import could not be found *([reportMissingImports](#pylance-diagnostics) or reportMissingModuleSource)*    | The package you are trying to import is not installed in the selected Python environment. | First, ensure the package is installed in your environment. If working in a `.py` file, type `python -m pip install {package_name}` in an activated terminal to install a package into your environment. For a Notebook, type `%pip install {package name}` in your code cell. If the package is installed, ensure you are using the correct environment/interpreter that matches the environment activated in your terminal. To select a new Python interpreter, use the **Python: Select Interpreter** command.|
```

Comment:
I think if we have a specific section for diagnostics, we can remove this from the table to avoid being repetitive. That way users will have a better understanding of how the docs are structured and can find all the information they need in one place.